nucypher/characters/lawful.py

```diff
@@ -800,7 +800,7 @@ def __init__(self,
 
                  # Blockchain
                  decentralized_identity_evidence: bytes = constants.NOT_SIGNED,
-                 checksum_address: str = None,  # Staker address
+                 checksum_address: str = None,  # TODO: Rename to Staker address
```

Comment:
I don't agree with marking this as a TODO, that would have lots of unexpected consequences, since Characters are assumed to have `checksum_address`. Instead, we could add a property `staker_address` to alias that.

Reply:
Let's have a discussion about this outside the scope of this PR.  I have a more broad solution in mind.

Reply:
Moved TODO. :heavy_check_mark: